code/upaya/app/views/devise/sessions/new.html.slim

```diff
@@ -0,0 +1,25 @@
+- title t('upaya.titles.visitors.index')
+
+.row
+  .panel.panel-default.same-height-panel
+    .panel-body
+      h2= t('upaya.headings.log_in')
+
+      = render 'devise/sessions/form'
+
+  .panel.panel-default.same-height-panel
+    .panel-body
+      h2= t('upaya.headings.visitors.new_account')
+      hr
+      p.disclaimer-basic= t('upaya.forms.session.need')
+      ul.not-menu =t('upaya.forms.session.need_list_html')
+      h5 =t('upaya.forms.session.advantage')
+      ul.not-menu =t('upaya.forms.session.advantage_list_html')
+
+      - if  AppSetting.registrations_enabled?
```

Comment:
I don't think this is an important feature but I suggest we keep it if it's more work to remove.